packages/contracts-bedrock/test/L2/SuperchainWETH.t.sol

```diff
@@ -174,7 +174,6 @@ contract SuperchainWETH_Test is CommonTest {
         // Check the total supply and balance of `_to` after the mint were updated correctly
         assertEq(superchainWeth.totalSupply(), _totalSupplyBefore + _amount);
         assertEq(superchainWeth.balanceOf(_to), _toBalanceBefore + _amount);
-        assertEq(superchainWeth.balanceOf(Predeploys.ETH_LIQUIDITY), 0);
```

Comment:
It seems a better fix be to add `vm.assume(_to != ETH_LIQUIDITY)`, so we can keep these assertions?

Reply:
We decided to remove this check as it doesn't make sense to include it. There's nothing in the contracts that prevents ETH_LIQUIDITY from being the receiver, so we shouldn't exclude that case in the tests. This assert should not be part of the test.

Reply:
that's fair, so maybe:
```solidity
uint256 expectedEthLiquidityBalance = _to != Predeploys.ETH_LIQUIDITY ? 0 : _toBalanceBefore + _amount;
assertEq(superchainWeth.balanceOf(Predeploys.ETH_LIQUIDITY), expectedEthLiquidityBalance);
```

Reply:
While this code would work, I still dont think this should be part of the test since the test should not care specifically about the superWeth balance of `ETH_LIQUIDITY` at any point.
The correct behavior of the ETH_LIQUIDITY contract is tested on `ETHLiquidity.t.sol` where it checks that the SafeSend works correctly and that the contract does not get any SuperWeth at any point.

Reply:
Not super opinionated here